packages/image_picker/image_picker_android/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java

```diff
@@ -79,6 +79,13 @@ String getPathFromUri(final Context context, final Uri uri) {
       // target file was written in full. Flushing the stream merely moves
       // the bytes into the OS, not necessarily to the file.
       return null;
+    } catch (SecurityException e) {
+      // Calling `ContentResolver#openInputStream()` has been reported to throw a
+      // `SecurityException` on some devices in certain circumstances. Instead of crashing, we
+      // return `null`.
+      //
+      // See [this issue](https://github.com/flutter/flutter/issues/100025) for more details.
```

Comment:
nit: the markdown formatting here won't do anything useful (as far as I know), I would just omit the `[this issue]` and the parens